etc/listproviders/acl.additional.actions.properties

```diff
@@ -7,3 +7,10 @@ list.name=ACL.ACTIONS
 
 # For example to set a custom action of type annotate with a label in the UI of Annotate.
 # Annotate=annotate
+
+# Allow users to simply use the annotation tool with a video,
+# i.e. to create their own annotations, tracks, categories, etc.
+Annotate=annotate
+# Allow users to manage the annotation tool environment for a video.
+# Also gives them access to other people's annotations.
+Manage\ annotations=annotate-admin
```

Comment:
I guess that was an accident?

Reply:
Yes, thanks. No need for Annotation Tool Actions here,